### PR TITLE
[IN-0]: Revert "[IN-2270] cocoapods 1.8 and up defaults to CDN (#348)"

### PR DESCRIPTION
**Pull request by peterheja-bitrise**
This reverts commit 3550c3038239d55fc1d44312d72b3f36a94d17ae.

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [ ] The tool I added is stable, and __does not require frequent updates__. _Preinstalled tools on the LTS stacks
  are not updated, so if the tool requires frequent updates you should handle the installation on-demand (see: [Install Any Additional Tool - DevCenter](https://bitrise-io.github.io/devcenter/tips-and-tricks/install-additional-tools/) for more information)_
- [ ] I updated the corresponding tests if there were any.
- [ ] I added a version report line to [`system_report.sh`](/system_report.sh) for the new tool(s) I added
- [x] I have added an entry to the CHANGELOG.md with a revisionID (the current date and the number of the change e.g.: 2019_10_11_1) and a brief description of the actual change
- [x] I have updated the revisionID in [bitrise_profile](roles/profiles/files/bitrise_profile) **BITRISE_OSX_STACK_REV_ID** with the one I just created in the CHANGELOG.md

### CODE REVIEW

**Comment by fehersanyi-bitrise** [comment]
but there is an error in the config file for some reason, I need to check

**Comment by peterheja-bitrise** [comment]
> but there is an error in the config file for some reason, I need to check

@fehersanyi-bitrise This is an exact revert of the original PR #348 , except the changelog and profiles.

**Comment by fehersanyi-bitrise** [comment]
@peterheja-bitrise sorry, commented on the wrong PR :D

**Review by fehersanyi-bitrise — APPROVED** [review]

File: roles/weekly-update-shared/tasks/main.yml [hunk 1]
```diff
@@ -44,6 +44,11 @@
 # --- pre-installed tool cache updates ---
 # ----------------------------------------
 
+- name: "CocoaPods: pod setup"
```
Comment (lmesz-bitrise) [inline roles/weekly-update-shared/tasks/main.yml:47]: Are we sure we want it back? The `ruby-gems` role is listed in the weekly-cache-update.playbook so the update will happen twice.

**Review by lmesz-bitrise — APPROVED** [review]

File: roles/weekly-update-shared/tasks/main.yml (hunk 1 above)
Comment (peterheja-bitrise) [reply roles/weekly-update-shared/tasks/main.yml:47]: I'd go with this for the time being. We can make it better in the future, though.

**Review by bitrise-tom — APPROVED** [review]

**Pull request merged**